profiler/src/ProfilerEngine/Datadog.Profiler.Native/StackSamplerLoopManager.cpp

```diff
@@ -194,83 +194,110 @@ void StackSamplerLoopManager::SendStatistics()
 
```

Comment:
todo: remove these changes.
I think it was just because it is impossible to recover from the deadlock situation